frontend/src/components/teamsAndOrgs/orgUsageLevel.js

```diff
@@ -6,15 +6,17 @@ import messages from './messages';
 import { StatsCardContent } from '../statsCardContent';
 import { ProgressBar } from '../progressBar';
 import { usePredictYearlyTasks } from '../../hooks/UsePredictYearlyTasks';
-import { useOrganisationLevel, useGetLevel } from '../../hooks/UseOrganisationLevel';
+import { useOrganisationLevel, usePredictLevel } from '../../hooks/UseOrganisationLevel';
 
 export function OrganisationUsageLevel({ completedActions, orgName, type, userIsOrgManager }) {
   const [currentLevel, nextLevelThreshold] = useOrganisationLevel(completedActions);
   const percent = parseInt((completedActions / nextLevelThreshold) * 100);
   const yearPrediction = usePredictYearlyTasks(completedActions, new Date());
-  const levelPrediction = useGetLevel(yearPrediction);
+  const levelPrediction = usePredictLevel(yearPrediction, type);
 
   const showTierInfo = ['DISCOUNTED', 'FULL_FEE'].includes(type) && userIsOrgManager;
+  const showDiscountLabel =
+    JSON.stringify(currentLevel) !== JSON.stringify(levelPrediction) && type === 'DISCOUNTED';
```

Comment:
What is the goal of that comparison between `currentLevel` and `levelPrediction`?

Reply:
If an organisation's tier did not change from its current level. Although, I should have tested it for different use cases. 🤔My only consideration was a free organisation whose predicted actions(<1000) would still maintain it at the same tier and so its cost would not really be discounted just maintained at 0.

Reply:
I think it would be good if we don't show the discounted tag if the predicted tier is FREE, so that should be the check.

Reply:
Alright